core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/BasicEntityRebindSupport.java

```diff
@@ -233,4 +237,14 @@ protected void addLocations(RebindContext rebindContext, EntityMemento memento)
             }
         }
     }
+    
```

Comment:
@bostko will address it in the next commit for separating rebind during provisioning from rebind during application starting. Merging now